Neos.Fusion/Resources/Private/Fusion/Prototypes/Component.fusion

```diff
@@ -0,0 +1,5 @@
+prototype(Neos.Fusion:Component) {
+  @class = 'Neos\\Fusion\\FusionObjects\\ComponentImplementation'
+  @sortProperties = false
+  renderer = ''
```

Comment:
we need to change that to `renderer = null`, as otherwise the path is existant and one doesnt get an error if its missed.

Reply:
will be fixed via https://github.com/neos/neos-development-collection/pull/3945